compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/NodeCloner.java

```diff
@@ -1311,6 +1312,20 @@ public void visit(BLangFromClause source) {
         clone.nillableResultType = source.nillableResultType;
     }
 
+    @Override
+    public void visit(BLangJoinClause source) {
+
```

Comment:
unnecessary new line at beginning and end. check other places as well

Reply:
Fixed in b0e02e2